src/pcapi/core/educational/repository.py

```diff
@@ -111,3 +111,7 @@ def find_educational_bookings_for_adage(
             )
 
     return educational_bookings_base_query.all()
+
+
+def find_redactor_by_email(redactor_email: str) -> Optional[EducationalRedactor]:
+    return EducationalRedactor.query.filter(EducationalRedactor.email == redactor_email).first()
```

Comment:
```suggestion
    return EducationalRedactor.query.filter(EducationalRedactor.email == redactor_email).one()
```

Si tu veux, vu qu'il est unique

Reply:
Il faut que je mette one_or_none du coup, mais oui, on peut avoir une 500 s'ils ne sont pas unique c'est mieux que de faile silencieusement :)